src/User/resources/i18n/ru/usuario.php

```diff
@@ -81,6 +81,7 @@
     'Account settings' => 'Настройки аккаунта',
     'Already registered? Sign in!' => 'Уже зарегистрированы? Войдите!',
     'An email has been sent with instructions for resetting your password' => 'Вам отправлено письмо с инструкциями по смене пароля',
+    'An email with instructions to create a new password has been sent to {email} if it is associated with an {appName} account. Your existing password has not been changed.' => 'Сообщение с инструкциями по созданию нового пароля было отправлено на адрес {email}, если он привязан к учетной записи {appName}. Ваш текущий пароль изменен не был.',
```

Comment:
I believe we will need to update all translation files.

Reply:
I not speak at 10 languages :cry:

Reply:
not translate...
run the ./yii message path/to/config.php command to update the new untranslated string to all of the language files.

Reply:
*That* update... OK I will try it now